catchemall/src/App.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import './App.css';
+import {BrowserRouter as Router, Route, Switch, Link} from 'react-router-dom';
+
+let pokeNumber = 1;
+
+const pokeTopTen = function () {
+  for (let i = 0; i < 10; i++) {
+    pokeNumber++;
+  }
+ } 
+
+const pokeNameGetter = function(poke) { 
+  return poke.forms[0].name;
+}
+
+
+class Pokemon extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: {},
+      isLoading: true,
+      hasError: false
+    }
+  }
+  componentDidMount() {
+    fetch(`https://pokeapi.co/api/v2/pokemon/${pokeNumber}`)
```

Comment:
I am not sure how you want to structure your app but to display all the poke names I changed your fetch to : fetch(`https://pokeapi.co/api/v2/pokemon`)